app/index.js

```diff
@@ -170,17 +155,54 @@ module.exports = yeoman.Base.extend({
   },
 
   askForLBVersion: function() {
+    var done = this.async();
+
+    var availableLBVersions;
+    Workspace.getAvailableLBVersions(function(err, LBVersions) {
+      if (err) {
+        return done(err);
+      } else {
+        availableLBVersions = Object.keys(LBVersions);
```

Comment:
This will run after you display the prompt. You should move `getAvailableLBVersions` to a standalone step.

In fact, we are already fetching them in `fetchLoopBackVersions` that set's `self.availableLBVersions`, see the lines before `askForLBVersion`.

Perhaps this is the result of resolving merge conflicts?